src/layouts/docs/docs-layout.module.css

```diff
@@ -25,31 +25,261 @@
   height: 100%;
   justify-content: center;
   overflow-y: auto;
-  padding: 24px 24px 100px 24px;
+  padding-bottom: 100px;
+  padding-left: 48px;
+  padding-right: 24px;
+  padding-top: 32px;
   position: relative;
 }
 
-.versionSwitcher {
-  height: 18px;
-  width: 346px;
-}
-
 /* TODO: these styles will be changing very soon, this is just to get us started */
 .main {
+  color: #3b3d45;
+  flex-grow: 1;
+  flex-shrink: 1;
+  font-size: 16px;
+  line-height: 22px;
   max-width: 804px;
   padding: 0 24px;
   padding-bottom: 50vh;
+  width: 0;
 
-  & h1:first-child {
-    margin-bottom: 12px;
+  & :global {
+    & a.__permalink-h {
+      display: none;
+    }
+
+    & .alert.alert-info {
+      background-color: #f2f8ff;
+      border: 1px solid #cce3f3;
+      border-radius: 6px;
+      color: #3b3d45;
+      padding: 24px;
+
+      & code {
+        background-color: white;
+        border: 1px solid #cce3fe;
+        border-radius: 5px;
+        color: inherit;
+      }
+
+      & p {
+        &:first-child {
+          margin-top: 0;
+        }
+
+        &:last-child {
+          margin-bottom: 0;
+        }
```

Comment:
What are these doing? Is it to avoid stacked margin from margin applied to `p`? Would margin collapsing come into play here? https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing

Reply:
So if these weren't here, then the alert boxes would be too tall like this:

![image](https://user-images.githubusercontent.com/43934258/147272867-9bb3265b-6f8f-4439-9611-abfd4e328c1c.png)

When these are present, the alert boxes look like this with the appropriate padding:

![image](https://user-images.githubusercontent.com/43934258/147272925-b4878500-b3b9-4cd2-a301-993ff9530c7f.png)

I'm not sure if we have any multi-paragraph alert boxes, but this approach allows multiple paragraphs to have the correct spacing between them without adding too much spacing at the top & bottom of the alert box. I can add a comment to explain since it is definitely interesting to see!

Reply:
Oh man I totally missed this was nested within the alert styles, my bad!

Reply:
No worries! 😊